roles/edpm_libvirt/tasks/configure.yml

```diff
@@ -1,5 +1,11 @@
 ---
 
+- name: Get libvirt password
+  ansible.builtin.set_fact:
+    sasl_password: "{{ lookup('file', edpm_libvirt_password_path, lstrip=false, rstrip=false) }}"
```

Comment:
we generally try to aovid the use of lookup plugigns as it generally not what you want

hum a lookup plugin always executes on the Ansible controller which is generally not what people want but I guess in this case this is appropriate.

this will execute the lookup on the filesystem of the AEE pod to which we would expect a secret to be mounted at /var/lib/openstack/configs/{{ edpm_service_name | default('libvirt') }}/LibvirtPassword

or /var/lib/openstack/configs/libvirt/LibvirtPassword in realitity since we should never have a custom libvirt service.